middleware/authz.go

```diff
@@ -0,0 +1,41 @@
+package middleware
+
+// This plugin is based on Casbin: an authorization library that supports ACL, RBAC, ABAC
+// View source at:
+// https://github.com/hsluoyz/casbin
+
+import (
+	"net/http"
+
+	"github.com/hsluoyz/casbin"
```

Comment:
@pkieltyka I believe that each directory in `github.com/pressly/middleware` could have its own `vendor/` deps.